src/contexts/Toast/ToastDemo.tsx

```diff
@@ -91,6 +94,12 @@ export const ToastDemo: React.FC<any> = () => {
               showToast(toastData)
             }}
           />
+          <Button
+            secondary={toasts.length > 0}
+            content="Clear all toasts"
+            disabled={toasts.length === 0}
+            onClick={clearAllToasts}
+          />
```

Comment:
Not particularly relevant, but you can play with the Toast notifications by going to `/dev/toast-demo`